.github/workflows/npm-brew.yaml

```diff
@@ -60,16 +58,10 @@ jobs:
       - name: Checkout code-server
         uses: actions/checkout@v3
 
-      - name: Checkout cdrci/homebrew-core
-        uses: actions/checkout@v3
-        with:
-          repository: cdrci/homebrew-core
-          path: homebrew-core
-
       - name: Configure git
         run: |
-          git config user.name github-actions
-          git config user.email github-actions@github.com
+          git config user.name cdrci
+          git config user.email opensource@coder.com
```

Comment:
Wasn't sure if I should use a real email or a fake one. Opted for a legit one in case something happened with this PR and someone needed to contact us 🤷🏼‍♂️